src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -118,11 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="ShellBannerNonWindowsPowerShell" xml:space="preserve">
-    <value>PowerShell {0}
-Copyright (c) Microsoft Corporation.
-
-https://aka.ms/powershell
-Type 'help' to get help.</value>
+    <value>PowerShell {0}</value>
```

Comment:
```suggestion
    <value>PowerShell {0}
    </value>
```

Reply:
I don't think we want a blank line after `PowerShell 7.3.0`.   But the changes you are suggesting are to provide just that.

Reply:
If so we need to remove `hostUI.WriteLine();`